tests/testthat/test-data_type.R

```diff
@@ -11,7 +11,7 @@ test_that("identify_data_type and split_raw_data works as expected", {
   
   df_layout_2 <- merge(df_layout, df_2)
   
-  df_merged_data <- gDRtestData::generate_response_data(df_layout_2, 0)
+  df_merged_data <- data.table::setDT(gDRtestData::generate_response_data(df_layout_2, 0))
```

Comment:
Will `gDRtestData` be also converted from `data.frame` to `data.table`? 👀 
If so...then it has to be reverted, right?

Reply:
fixed